x/posts/keeper/posts.go

```diff
@@ -0,0 +1,174 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	"github.com/desmos-labs/desmos/v3/x/posts/types"
+)
+
+// SetPostID sets the new post id for the given subspace to the store
+func (k Keeper) SetPostID(ctx sdk.Context, subspaceID uint64, postID uint64) {
+	store := ctx.KVStore(k.storeKey)
+	store.Set(types.PostIDStoreKey(subspaceID), types.GetPostIDBytes(postID))
+}
+
+// GetPostID gets the highest post id for the given subspace
+func (k Keeper) GetPostID(ctx sdk.Context, subspaceID uint64) (postID uint64, err error) {
+	store := ctx.KVStore(k.storeKey)
+	bz := store.Get(types.PostIDStoreKey(subspaceID))
+	if bz == nil {
+		return 0, sdkerrors.Wrapf(types.ErrInvalidGenesis, "initial post ID hasn't been set for subspace %d", subspaceID)
+	}
+
+	postID = types.GetPostIDFromBytes(bz)
+	return postID, nil
+}
+
+// DeletePostID removes the post id key for the given subspace
+func (k Keeper) DeletePostID(ctx sdk.Context, subspaceID uint64) {
```

Comment:
I agree the naming is fancy, but it reflects the associated ones (`SetPostID` and `GetPostID`). I couldn't find any better alternative, but I'm open to suggestions

Reply:
I thin the `SetPostID` name is fine, but we can enhance the other two.
What about:
- `GetLastPostID` (Since the function is retrieving the last add post ID)
- `DeleteLastPostID` (Since the function is deleting the last added post ID)
- 
They will be more verbose but at least it will be clear by reading them what they're doing.
And I think this will ease devs job when looking at the code

Reply:
To make everything even more clear I've just renamed all those methods to be `[Set/Get/Delete]NextPostID` and the attachment ones to be `[Set/Has/Get/Delete]NextAttachmentID` as this should be even better to understand properly what they are related to